README.md

```diff
@@ -3,33 +3,29 @@ This is the source code for <http://tutorials.codebar.io>
 ## Getting started
 
 This is a [GitHub Pages](https://pages.github.com/) repo, so you can render the pages with [Jekyll](http://jekyllrb.com/).
+All you need is to make sure you have installed the version of Ruby indicated in
```

Comment:
Can we please amend the wording of this as installing Ruby isn't always easy and using wording like "All you need is to make sure" makes it sound like a much simpler task than what it could end up being.